### ISSUE TITLE
Missing parameter

### ISSUE THREAD

**Issue by satr** [Azure/secrets-store-csi-driver-provider-azure#1104]
useVMManagedIdentity: "false"

Step 5 in the [Configure workload identity](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-identity-access#configure-workload-identity)

**Comment by aramase** [comment]
@satr what is the question? `useVMManagedIdentity` is not required when using workload identity.

**Comment by satr** [comment]
In [documentation](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-identity-access#configure-workload-identity) it was set and connection did not work for me without this property, when I tested this. After setting this property (as it was shown in the document), it start working. Now it works without this property (after your comment I changed the class creation logic), but I'm not sure if this is not needed in all conditions. I've seen in the git, its default value is false. If you confirm it is not needed, the issue can be closed, I guess.
Thank you.
